baybe/surrogates/gaussian_process/presets/default.py

```diff
@@ -17,54 +19,39 @@
     from baybe.kernels.base import Kernel
     from baybe.searchspace.core import SearchSpace
 
+# Boundaries for low and high dimension limits
```

Comment:
In the remaining code base, we put these as docstrings below the constants

Reply:
ok, changed
why do we do that again?

Reply:
I think
* For consistency (in our classes, we also put the docstring below attributes)
* When they are public, they will actually appear in the doc, I think

Reply:
Just noticed:
* The "comment" below the variable should really be a docstring, i.e. using triplet quotes
* Please add the note about the average also to the noise factory, because its an isolated piece of code

Reply:
I'm not sure you're right about the constants, where do we actually do this? This case here is `private` and `not a class attribute`, so why would anything you said apply? In fact looking for isntance here 
![image](https://github.com/user-attachments/assets/02598013-4829-4d9a-8507-8efa402c57f6)
 I dont see the comment i) below ii) in triple quotes. Why did you suggest this? I think I should actually reverse it and have the comment above for consistency

Added the comment to the noise factory

Reply:
The case you showed is an exception since not user-facing and indeed part of a private package `_optional` that does not appear in the docs. And the thing put as comment is not describing the individual meaning of these constants but rather takes the role as a heading for a code block.

We do in fact have a few rare examples, like `Smiles`:
https://emdgroup.github.io/baybe/_autosummary/baybe.parameters.substance.html#module-baybe.parameters.substance

As you can see, the point is that things will get rendered in the doc. So even if the variable in your case is private, it makes sense to use a docstring:
* for consistency
* because it's 100% guaranteed that we would forget to turn the comment into a docstring in cases where we turn a private constant in a public one during refactoring

Reply:
I see that you've simply merged without actually having us resolved the issue 😞 ...

Agree that this erroneously public and I'm not sure if there are other actual examples in the current doc, but that is not the point of my argument! The rationale is simply: Everything public needs a docstring. Why use different rules for private objects? Makes it more complicated + they can become public in the future! Also, according to your logic, we should also put no docstrings under our private attributes!

### Your logic (not consistent, requires changes during refactoring)
```python
PUBLIC_CONSTANT = 1
"""Here you agree with docstring."""

# Following your logic, i.e. since not rendered, we'd need to put a comment here. 
# As soon as we make this value public, though, we need to turn the comment into a docstring,
# with additional effort and high chance of forgetting
_PRIVATE_CONSTANT = 1

@define
class A:
    public_attr = 0
    """Here you agree with docstring."""

    # Following your logic, i.e. since not rendered, we'd need to put a comment here. 
    # As soon as we make this value public, though, we need to turn the comment into a docstring,
    # with additional effort and high chance of forgetting
    _private_attr = 0
```
 
### My logic (everything consistent, no need for any changes)
```python
PUBLIC_CONSTANT = 1
"""Docstring."""

_PRIVATE_CONSTANT = 1
"""Docstring."""

@define
class A:
    public_attr = 0
    """Docstring."""

    _private_attr = 0
    """Docstring."""
```